game_app.rb

```diff
@@ -0,0 +1,131 @@
+require 'json'
+require_relative './gameAuthor/game'
+require_relative './gameAuthor/author'
+
+class GameApp
+  attr_accessor :games, :authors, :item
+
+  def initialize
+    @games = []
+    @authors = []
+    @path = './storage/games.json'
+    load_data
+  end
+
+  def get_input(prompt)
+    print prompt
+    gets.chomp
+  end
+
+  def game_inputs
+    publish_date = get_input('Publish date(YYYY/MM): ')
+    multiplayer = get_input('Is it a multiplayer game?: ')
+    last_played_at = get_input('When was the game last played?(YYYY/MM): ')
+    {
+      publish_date: publish_date,
+      multiplayer: multiplayer,
+      last_played_at: last_played_at
+    }
+  end
+
+  def author_inputs
+    first_name = get_input('Insert authors first name: ')
+    last_name = get_input('Insert authors last name: ')
+    {
+      first_name: first_name,
+      last_name: last_name
+    }
+  end
+
+  def create_game
+    inputs = game_inputs
+    Game.new(inputs[:publish_date], inputs[:multiplayer], inputs[:last_played_at])
+  end
+
+  def create_author
+    inputs = author_inputs
+    Author.new(inputs[:first_name], inputs[:last_name])
+  end
+
+  def select_author
+    puts 'Select an option for the author of the game'
+    puts '0) Create a new Author'
+    @authors.each_with_index do |author, index|
+      puts "#{index + 1}) #{author.first_name} #{author.last_name} "
+    end
+    gets.chomp.to_i
+  end
+
+  def add_game
+    game = create_game
+    selected = select_author
+    author = if selected.zero?
+               create_author
+             else
+               @authors[selected - 1]
+             end
+    add_author_game(author, game)
+  end
```

Comment:
- [ ] @team great work adding UI for adding and displaying games, however when I add a game it doesn't tell me if it's stored or not, I have to go to the menu to check if it's stored, it would be nice if you added a message to tell the user when a game and author are stored as you did for album and book.
![image](https://github.com/jeanbuhendwa/catalog-of-my-things/assets/61408860/84d700d0-5505-4d8b-9c7b-53caab868042)

Reply:
- [ ] @team kindly ensure the game are stored even after closing the app, I got the error below on exit and the games and authors are not stored in a JSON file, it seems like the file referenced doesn't exist
![image](https://github.com/jeanbuhendwa/catalog-of-my-things/assets/61408860/3a2490ef-e000-4060-bdbf-f3125288c1bd)